tests/manage/test_create_storageclass_with_wrong_provisioner.py

```diff
@@ -0,0 +1,100 @@
+import pytest
+import logging
+
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.framework.testlib import ManageTest, tier3
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier3
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+)
+class TestCreateStorageClassWithWrongProvisioner(ManageTest):
+    """
+    Create Storage Class with wrong provisioner
+    """
+    @pytest.mark.parametrize(
+        argnames="interface",
+        argvalues=[
+            pytest.param(
+                *["RBD"], marks=pytest.mark.polarion_id("OCS-620")
+            ),
+            pytest.param(
+                *["CEPHFS"], marks=pytest.mark.polarion_id("OCS-621")
+            )
+        ]
+    )
+    def test_create_storage_class_with_wrong_provisioner(self, interface):
+        """
+         Test function which creates Storage Class with
+         wrong provisioner and verifies PVC status
+        """
+        log.info(f"Creating a {interface} storage class")
+        self.sc_data = templating.load_yaml_to_dict(
```

Comment:
Please use helpers.create_storage_class and allow this function to get the provisioner

Reply:
Done